### DISCUSSION TITLE
Spam Emails sent from Unknown Credentials

### DISCUSSION THREAD

**Discussion by JAP42** [postalserver/postal#2018] [Help with using Postal] [answered]
I have Spam emails being sent from my server. But they show up in the outgoing messages under unknown credentials. The only open port on my postal docker is 25, Even the web server is only exposed behind a reverse proxy. Any idea what going on here? My legitimate emails show the proper creds.

**Comment by willpower232** [comment, marked as answer]
I think this happens when the send message form is used in the UI.

![image](https://user-images.githubusercontent.com/1619102/173147091-22f4d34e-5266-4441-838b-9f201b0a6309.png)

It might be worth resetting all user passwords and ensuring that no one has a login that shouldn't

**Reply by JAP42** [reply]
I was thinking web UI and did change the password. I have not seen one for awhile, we will see but I think this was probably it.

**Reply by willpower232** [reply]
definitely worth keeping an eye on, you can also have a look at the spam messages as the received header section will tell you the IP address the message came from
